### PR TITLE
Created dummy children route

### PR DESCRIPTION
**Pull request by thrvg**
## Notion ticket link

[Add dummy create /children route](https://www.notion.so/uwblueprintexecs/134e1f159caa4f73939d11a700c325ad?v=39cf7d5a98384eb9af2bff2265d0ac8a&p=654486bc02724cd6bed1cb28fa2a2290)

## Implementation description
* Created a new app/rest/child_routes.py file in order to create the children route

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: backend/python/app/rest/child_routes.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from flask import Blueprint, current_app, jsonify, request
+
+from ..middlewares.auth import require_authorization_by_role
+from ..middlewares.validate import validate_request
+from ..resources.child_dto import ChildDTO
+from ..services.implementations.child_service import ChildService
+
+child_service = ChildService(current_app.logger)
+
+
```
Comment (jennchenn) [inline backend/python/app/rest/child_routes.py:12]: nit remove extra line

File: backend/python/app/rest/child_routes.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+blueprint = Blueprint("child", __name__, url_prefix="/children")
+
+
```
Comment (jennchenn) [inline backend/python/app/rest/child_routes.py:15]: nit remove extra line

**Review by jennchenn — APPROVED** [review]
looks good! nice work 🙌

**Comment by thrvg** [comment]
One of the empty lines was apparently a linter thing. The linter thinks that's the best format.

**Pull request merged**